.github/workflows/nightly.yml

```
@@ -96,6 +97,7 @@ jobs:
           path: apps/desktop/upload
 
   nightly-server:
+    if: github.repository == 'TriliumNext/Trilium'  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)